kafka-gateway/README.md

```diff
@@ -18,9 +18,47 @@ example properties provided via VM options
 -Dspline.kafka.topic=spline-topic
 ```
 
-### Error recovery
-Currently, all received messages are acknowledged. Errors during message processing are logged. 
-The consumer is idempotent so if needed it can be rewind back, and the failed messages can be re-ingested. 
+### Idempotency
+In accord with Kafka best practices this consumer is indepmotent. 
+In practice, it means that one message can be consumed multiple times without risk of duplication. 
+Spline will recognize that the message was already consumed, and it will simply ignore it.
+
+### Scalability
+By default, Spline Consumer is using one Kafka Consumer. When need to scale arise you can run additional Spline Consumer instances.
```

Comment:
```suggestion
By default, Spline Consumer uses a single Kafka Consumer. When a need to scale arises you can run additional Spline Consumer instances.
```

Also, why is it called a "Spline Consumer". In our terminology "Spline Consumer" is another module, and I think it creates a confusion. Should we call it just Spline (without a word "consumer")?